cirrus/cirrus-qa.sh

```diff
@@ -4,15 +4,12 @@
 source cirrus-env QA
 source set_maven_build_version $BUILD_NUMBER
 
+
 # Install a specific version of Maven. We want to test against multiple versions.
 MAVEN_HOME_IT=`pwd`/maven_it
 mkdir -p $MAVEN_HOME_IT
 curl -sSL https://repo1.maven.org/maven2/org/apache/maven/apache-maven/$MAVEN_VERSION/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar zx --strip-components 1 -C $MAVEN_HOME_IT
 cp -f ~/.m2/settings.xml $MAVEN_HOME_IT/conf/
 
-# We need to build this small plugin first, that will dump the analysis properties in a local file
-mvn -f property-dump-plugin/pom.xml --batch-mode -Dmaven.home=$MAVEN_HOME_IT install
-
-# Run ITs.
-cd its
-mvn -B -e -Dsonar.runtimeVersion=$SQ_VERSION -Dmaven.test.redirectTestOutputToFile=false -Dmaven.home=$MAVEN_HOME_IT verify
+# we don't need to rebuild the sonar-maven-plugin, it is already deployed on repox
+mvn -B -e -pl '!sonar-maven-plugin' -Pits -Dsonar.runtimeVersion=$SQ_VERSION -Dmaven.test.redirectTestOutputToFile=false -Dmaven.home=$MAVEN_HOME_IT verify
```

Comment:
Why not take this occasion to use long options for the tests?
```suggestion
mvn --batch-mode --errors --projects '!sonar-maven-plugin' --activate-profiles its -Dsonar.runtimeVersion=$SQ_VERSION -Dmaven.test.redirectTestOutputToFile=false -Dmaven.home=$MAVEN_HOME_IT verify
```

Reply:
:+1: